### PR TITLE
feat: Botkit tests created

### PR DESCRIPTION
**Pull request by ColorfulCodes**
Here are the tests using Botkit. One file is the test and the other is being tested.

### CODE REVIEW

File: skills/project-command.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//  command: '/project',
+//  text: 'add google google.com'
+//  };
+//
+// let exampleInput2 = {
+//  channel_Name: 'int-project-bot',
+//  channel_id: 'c9367a',
+//  command: '/project',
+//  text: 'Add colby-portfolio colbyallen.co'
+// };
```
Comment (nrambeck) [inline skills/project-command.js]: Let's delete the comments from this file.

File: test/controllers/add.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const Botmock = require('botkit-mock');
+const {expect, assert} = require("chai");
```
Comment (nrambeck) [inline test/controllers/add.test.js]: We are only using the `expect` function down below, so let's drop the `assert` function from being imported.

File: skills/project-command.js (hunk 1 above)
Comment (ColorfulCodes) [reply skills/project-command.js]: Will do.

File: test/controllers/add.test.js (hunk 2 above)
Comment (ColorfulCodes) [reply test/controllers/add.test.js]: no prob.

**Comment by nrambeck** [comment]
@ColorfulCodes, almost there, but not quite. I tried pulling this down and running `npm test`, but it says `Error: not test files found`. I think we are missing the changes we made to the `package.json` file to search inside directories for tests.

**Comment by nrambeck** [comment]
The test script now runs, but it fails because we are missing the `botkit-mock` dependency. That should be committed to the `package.json` under `devDependencies`.

**Pull request merged**